prac_06/programming_language.py

```diff
@@ -0,0 +1,40 @@
+"""
+CP1404 Practical 06 - Programming Language class
+Time estimated: 30 minutes
+Time taken: 28 minutes
+"""
+
+
+class ProgrammingLanguage:
+    """Represent Programming Language object."""
+
+    def __init__(self, name="", typing="", reflection="", year=0):
```

Comment:
For reflection maybe it should be a boolean (True or False) as in the table its either a 'yes' or 'no' for reflection. Also in the tests it shows that for reflection its its either True or False so maybe you should be expecting a boolean instead of a string.